pkg/gitops/GitOpsConfigService.go

```diff
@@ -126,6 +126,15 @@ func NewGitOpsConfigServiceImpl(Logger *zap.SugaredLogger, ciHandler pipeline.Ci
 
 func (impl *GitOpsConfigServiceImpl) ValidateAndCreateGitOpsConfig(config *bean2.GitOpsConfigDto) (DetailedErrorGitOpsConfigResponse, error) {
 	detailedErrorGitOpsConfigResponse := impl.GitOpsValidateDryRun(config)
+	//hostUrl := config.Host
```

Comment:
remove unused code

Reply:
this was not written by me ,this is merged from devtron:main